user/pages/02.development/03.creating-a-skill/docs.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Development Setup
+taxonomy:
+    category:
+        - docs
+---
+
+# Setting up the folders
+
+Once you've come up with your idea for a skill, make a new folder in `mycroft-core/mycroft/skills` with your desired skill name.
+
+For skills, there should generally be three things inside this folder - a folder called `dialog`, a file called `__init__.py`, and a folder called `vocab`. We'll go through these one by one so you know why each is there.
+
+###### Dialog
+The `dialog` folder contains subfolders for each spoken language the skill supports, each of which has `.dialog` files for what Mycroft should say when he executes a skill.
+
+For my welcome skill, I create the `dialog` folder. Inside, I create the subfolder `en-us`, which corresponds to American English. This folder should contain .dialog files, which will determine what Mycroft will say.

```

Comment:
Let's use third person for the whole documentation and add links to refer examples. 
We can rephrase that sentence with something like:

Let's take a look at [Welcome Skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/welcome) as an example. It contains a `dialog` and `vocab` folder which both of them contains a subfolder `en-us` corresponding to American English. ... ... ...